apps/prairielearn/src/models/course-request.sql

```diff
@@ -69,6 +69,12 @@ WITH
       INNER JOIN users AS u ON u.user_id = r.user_id
       LEFT JOIN users AS ua on ua.user_id = r.approved_by
       LEFT JOIN select_course_request_jobs AS j ON j.id = r.id
+    WHERE
+      (
+        $show_all != 'true'
+        AND r.approved_status NOT IN ('approved', 'denied')
+      )
+      OR ($show_all = 'true')
```

Comment:
A slightly improved way of writing this would be:

```suggestion
      CASE
        WHEN $show_all = TRUE THEN TRUE
        ELSE r.approved_status NOT IN ('approved', 'denied')
      END
```

Reply:
This is outdated; we used Jonatan's suggestion instead.